string name
string motion
uint8 type
---
